File: src/components/Message.js
export const Message = ({message}) => {

    let messageStyle = "message";
    return (
        <div className={messageStyle}>

            {/* <div className="owner">
                {message.sender.username}
            </div> */}

            <div className="text">
                {message.text}
            </div>

            <div className="time">
                {message.timeSent}
            </div>
        </div>
    );
}

File: server/Server/Server/Models/Message.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.IO.Compression;

namespace Server.Models
{
    public class Message
    {
        [Key]
        public int Id { get; set; }
        
        public string Text { get; set; }

        public User Sender { get; set; } = new User();

        public string TimeSent { get; set; }

        [ForeignKey("ChatId")]
        public Chat Chat { get; set; } = new Chat();

        public bool IsEdited { get; set; }

        // public List<Emotion> Emotions { get; set; } = new List<Emotion>();
    }
}
